Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertDirectShape.cs

```diff
@@ -230,8 +233,11 @@ public ApplicationObject DirectShapeToNative(DirectShape speckleDs, ToNativeMesh
     }
     catch (Exception ex)
     {
+      // TODO : check if catch block is necessary
```

Comment:
None of these conversions should be catching Exception, so we should at least have the !IsFatal() pattern and the warning should not be disabled. Reserving judgement on LogDefaultError() but it would be good to know WHY we throw exceptions here as we can then provide a better log message.

I won;t keep logging the same for these conversisons, but it stants for each place - we need to use the !IsFatal() pattern here, log, and not suppress the warning.